src/solar_balancer

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python2
+#!/usr/bin/env python3
```

Comment:
I think we should stick with direct path to `/usr/bin/python3`, not via `env`, as elsewhere in the code base, to use the system/platform python installation consistently rather than the current user's:

```
$ git grep python3
src/Makefile:# (Both must be python3, python2 is no longer supported)
src/Makefile:SC_PYTHON  := /usr/bin/python3
src/Makefile:SC_PYTHON_CONFIG := /usr/bin/python3-config
src/core/sc_python.c:/* Python2 implemented this as PyString_AsString, for python3 we have to go
src/solar_capture:#!/usr/bin/python3
src/solar_capture:# Currently solar_capture requires python3
src/solar_capture_doc:#!/usr/bin/python3
src/solar_capture_monitor:#!/usr/bin/python3
```
I am aware there are a few other cases in the code base that also need fixing up!

Reply:
Happy to stick with your standard.